Kernel/FileSystem/Inode.h

```diff
@@ -36,7 +36,6 @@ class Inode : public ListedRefCounted<Inode, LockType::Spinlock>
 
 public:
     virtual ~Inode();
-
```

Comment:
nit: pointless whitespace change, we usually separate ctors/dtors from everything else like that.